examples/crossplane/values.yaml

```diff
@@ -0,0 +1,5 @@
+nodeSelector:
```

Comment:
same as default yaml right? Why include in example?

Reply:
done